src/scss/components/carousel.scss

```diff
@@ -158,12 +165,14 @@
   position: static;
   width: auto;
   margin: 0;
-  padding: 7px 0 7px 14px;
+  padding-left: 14px;
 
   li,
   li.active {
+    display: inline-block;
     margin-right: 4px;
     margin-left: 4px;
+    vertical-align: middle;
 
     &:hover {
       border-color: rgba($carousel-caption-color, .8);
```

Comment:
Not related to this line, but for the carousel-mask below the mask colour is not coming out the same as before - is that intended?
It is down to the '$carousel-mask-bg-color' variable in variables.scss being different based on black rather than the theme colour. Might need to change it to 
`$carousel-mask-bg-color:                      rgba($theme-bg-lighter, 0.55) !default;`

Reply:
Done! Although there was a note above that line in `variables` where the variable was defined, talking about accessibility colour contrasts. Not sure sure if that was why the change was originally made . . .

`Meets 3:1 for large text (if using small text this should meet contrast of 4.5:1 to $carousel-text-color)`

Reply:
Sorry, my mistake with this one, I didn't realise it had been changed prior to our upgrade work, in which case we should probably leave this as it was. Is it OK to change this back to rgba(0, 0, 0, 0.4) !default;

Reply:
Done!